app/routes/_layout+/assets._index.tsx

```diff
@@ -112,25 +112,25 @@ export async function loader({ context, request }: LoaderFunctionArgs) {
 
     return mode === "SIMPLE"
       ? await simpleModeLoader({
-          request,
-          userId,
-          organizationId,
-          organizations,
-          role,
-          currentOrganization,
-          user,
-          settings,
-        })
+        request,
```

Comment:
I think something is happening with your prettier. Can you please double check it so we dont get those formatting issues?